using-timescaledb/continuous-aggregates.md

```diff
@@ -353,47 +310,26 @@ continuous aggregate, write:
 CALL refresh_continuous_aggregate('conditions_summary_hourly, NULL, NULL);
 ```
 
-We do not recommend doing so for tables that see continuous ingest of
-new data since that would trigger a refresh of buckets that are not
-yet filled completely.
-
-However, note that this might materialize a lot of data, can affect
-other policies such as the data retention, and affects write
-amplification, so it should only be used rarely.
+However, we do not recommend open-ended refreshes on continuous
+aggregates when there is a continuous ingest of new data since that
+would trigger a refresh of time buckets that are not yet completely
+filled. It might also materialize a lot of data, increase write
+amplification, and affect other policies such as data retention.
 
 >:TIP: You should avoid refreshing time intervals that still see a lot
 >of writes, which is usually the last bucket of the continuous
->aggregate. These intervals are still changing and will not produce
->accurate aggregate anyway and refreshing unnecessarily will increase
->the write amplification, which will slow down the ingest rate of the
->hypertable. If you want to ensure that you read the latest bucket, you
->should instead rely on [real-time aggregates][real-time-aggregates].
+>aggregate. These intervals are still changing and are unlikely to
+>produce accurate aggregates, while at the same time slowing down the
+>ingest rate of the hypertable due to write amplification. If you want
+>to include the latest bucket in your queries, you should instead rely
+>on [real-time aggregation][real-time-aggregates].
 
 The `schedule_interval` option to `add_continuous_aggregate_policy`
-controls how frequently materialization jobs will be launched. Setting
-a shorter interval will mean materializations happen more frequently
-but each job consumes background worker resources while it is running.
-
-#### Disabling real-time aggregates
-
-When querying the continuous aggregate view (for example,
-`conditions_summary_hourly`) by default you will get a complete view of
-the data including both the partially aggregated data and newer,
-unaggregated data in the hypertable (for example, `conditions`),
-based on the [real-time aggregate capability](#real-time-aggregate).
-
-If you, however, want to just get partially aggregated data and not
-include recent data that has yet to be materialized, you can set the
-option `timescaledb.materialized_only` to `true` using [`ALTER
-MATERIALIZED VIEW`][api-alter-cagg]:
-
-```sql
-ALTER MATERIALIZED VIEW conditions_summary_hourly SET (
-    timescaledb.materialized_only = true
-);
-```
+controls how frequently materialization jobs will be run. Setting a
+shorter interval will refresh more frequently but at the same time
+consume more background worker resources.
 
-**Using `timescaledb.information` Views:**
+#### Using `timescaledb.information` Views
```

Comment:
Aren't there more of these in the document?

Reply:
I think so, but I changed this because the content there didn't fit within the current section which is about real-time aggregation. So, it needs to be its own subsection.

Reply:
I'm only worried about consistency. Are there other sections that use `**whatever**` style header that need to be changed to `####whatever`?